code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -318,6 +323,8 @@
 					enemy_tile.consider_pressure_difference(src, -difference)
 			//This acts effectivly as a very slow timer, the max deltas of the group will slowly lower until it breaksdown, they then pop up a bit, and fall back down until irrelevant
 			LAST_SHARE_CHECK
+		if(active_hotspot && enemy_tile.active_hotspot)
+			our_hot_group.merge_hot_groups(enemy_tile.our_hot_group)
```

Comment:
this follows back on what I posted before, should hotspots manage the group rather then the turf?